orchagent/portsorch.cpp

```diff
@@ -1414,7 +1468,11 @@ bool PortsOrch::setPortAdminStatus(Port &port, bool state)
         SWSS_LOG_ERROR("Failed to set admin status %s for port %s."
                        " Setting host_tx_ready as false",
                        state ? "UP" : "DOWN", port.m_alias.c_str());
-        m_portStateTable.hset(port.m_alias, "host_tx_ready", "false");
+
+        if (!m_cmisModuleAsicSyncSupported)
+        {
+            m_portStateTable.hset(port.m_alias, "host_tx_ready", "false");
```

Comment:
@noaOrMlnx can we use the newly introduced API setHostTxReady() here as well instead of directly calling hset?

Reply:
Done